model/src/main/kotlin/ScanResults.kt

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright (c) 2017-2018 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.model
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.databind.JsonNode
+
+import java.time.Instant
+import java.util.SortedSet
+
+/**
+ * A list of [ScanResult]s for the package identified by [id].
+ */
+data class ScanResults(
```

Comment:
I wonder whether we should name this class differently. I find it confusing that the below `val results: List<ScanResult>` is also result*s* (plural). In general, the `ScanResults` and `ScanResult` might be easily confused. So maybe call this class something like `ScanReport`?

Reply:
`ScanReport` sounds like a single report to me, `ScanResults` better describes the content of the class. Let's discuss the naming offline.

Reply:
As discussed, using `ScanResultContainer` now. Was thinking about `ScanResultsContainer` but then decided to use the singular like e.g. Gradle does for container classes.